README.md

```diff
@@ -52,7 +52,7 @@ nexus and to the maven repo (last is only possibly from master).
 ## Release Notes
 
 ### 0.3.0
-- To fill
+- Add POI Gravitation endpoint
```

Comment:
this seems to also include that you can use geometries as source? is this an extra MR that should be reviewed before or together? if together can you please add this to the release notes?

Reply:
No, using geometries as source was already included before